build_tools/provision_machine.py

```diff
@@ -40,6 +40,7 @@
 from _therock_utils.artifacts import ArtifactPopulator
 import requests
 from packaging.version import Version, InvalidVersion
+import subprocess
```

Comment:
nit: please sort imports alphabetically.

I don't have a strong opinion on grouping of `import *` and `from * import *` lines. Some files mix them together, some put all of one type first.